mmv1/products/osconfig/api.yaml

```diff
@@ -401,19 +397,11 @@ objects:
                   - :UPDATE
              - !ruby/object:Api::Type::Array
               name: 'excludes'
-              exactly_one_of:
-                - patch_config.0.windows_update.0.classifications
-                - patch_config.0.windows_update.0.excludes
-                - patch_config.0.windows_update.0.exclusive_patches
               description: |
                 List of KBs to exclude from update.
               item_type: Api::Type::String
              - !ruby/object:Api::Type::Array
               name: 'exclusivePatches'
```

Comment:
Can we also add `exclusive_patches` to the test to make sure these three fields do not conflict with each other?